ui/src/assets/docs/de/faq-page.md

```diff
@@ -0,0 +1,136 @@
+# Frequently Asked Questions
+
+---
+
+## Consents
+
+**Warum kann ich keinen Consent anlegen?**\
+Die Rolle in Ihrem QWAC Zertifikat beinhaltet die benötigte Rolle AIS nicht.
+In diesem Fall sollte Ihnen folgender Fehler angezeigt werden:
+
+```
+{
+ "timestamp": 1549441548991,
+ "status": 403,
+ "error": "Forbidden",
+ "message": "You don't have access to this resource",
+ "path": "/v1/consents"
+}
+```
+
+Lösung: Entweder Sie erstellen sich mittels des Zertifikatsservices ein neues Zertifikat mit der benötigten Rolle AIS und integrieren es anschließend in Ihren Request oder Sie entschlüsseln das Zertifikat und prüfen, ob "PSP_AI" beinhaltet ist.
+
+Ein anderer Grund könnte sein, dass das Attribut "recurringIndicator" in Ihrem Request auf "false" gesetzt ist und das Attribut "frequencyPerDay" ungleich "1" gesetzt wurde. Der Standardwert für das Attribut "recurringIndicator" sollte auf "true" gesetzt werden, da ein Drittanbieter die Kontodaten in der angegebenen Zeit abfragen können möchte. 
```

Comment:
Eine andere Fehlerursache könnte sein, dass das Attribut "recurringIndicator" in Ihrem Request auf "false" und das Attribut "frequencyPerDay" ungleich "1" gesetzt ist. Standardmäßig ist das Attribut "recurringIndicator" auf "true" gesetzt, da ein Drittanbieter die Kontodaten in der angegebenen Zeit abfragen möchte.

Reply:
done